inc/agent.class.php

```diff
@@ -823,7 +823,7 @@ public function getSearchOptionsNew() {
          'table'              => $this->getTable(),
          'field'              => 'mdm_type',
          'name'               => __('MDM type', 'flyvemdm'),
-         'datatype'           => 'bool',
+         'datatype'           => 'specific',
```

Comment:
If you set ```specific``` here you must update getSpecificValueToDisplay(), and use getEnumMdmType() to map the value of the field into a localized equivalent.